modules/op-optional-configuration-options.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: REFERENCE
+[id="op-optional-configuration-options_{context}"]
+= Optional configuration options
+
+[role="_abstract"]
+You can configure the following fields that do not have a default value and thus are considered only if you pass them. By default, the Operator does not add and configure these options in the `TektonConfig` CR.
+
+* `default-timeout-minutes`: This option sets the default number of minutes to use for `TaskRun` and `PipelineRun`, if none is specified. For example, `default-timeout-minutes: 60` sets 60 minutes as default.
```

Comment:
@vdemeester Can you please confirm?

Reply:
This does set the default timeout for TaskRun and PipelineRun. This means that if it's set to 60min and the user doesn't specify any timeout when creating a TaskRun or PipelineRun, it would default to 60min, meaning a TaskRun or a PipelineRun taking more than 60min to complete, would time-out and be cancelled.

Reply:
@vdemeester Thank you for clarifying this further. @ekristova Could you please restructure the description for better clarity? The current description only conveys the name of the field, which is anyhow clear. What is the intent of setting this default timeout needs to be clarified?